_posts/2018-08-XX-custom-resources-in-0.5.0.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title:  "Strimzi adopts Custom Resources"
+date: 2018-08-XX
+author: tom bentley`
```

Comment:
You have to add your self to the [authors.yml file](https://github.com/strimzi/strimzi.github.io/blob/master/_data/authors.yml). And I'm not sure how it will deal with the space.

Reply:
@tombentley I have added you as authour with name "tom_bentley" in the authors YAML file. It's on the master so you can rebase and use it.